### PR TITLE
setPixelRatio() calls setSize()

### PR DESCRIPTION
**Pull request by BenediktS**
If we set a new pixelRatio, we should reset the size of the canvas too.

### CODE REVIEW

**Comment by BenediktS** [comment]
After rereading #6027, #6028 i am a little bit confused.
Are you planning to delete the pixelRatio or not? 

The pixelRatio is a very easy tool to render different qualities with only one parameter.
I think if someone is useing the pixelRatio parameter, he/she is in a "autosize canvas" mode.

File: src/renderers/WebGLRenderer.js [hunk 1]
```diff
@@ -476,6 +476,8 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 		pixelRatio = value;
 
+		this.setSize( _width, _height, true );
```
Comment (mrdoob) [inline src/renderers/WebGLRenderer.js]: Forcing `true` here may have side effects...

**Comment by BenediktS** [comment]
changed it to "false". 

You are right, updateStyle is only needed when you set a new size.

**Comment by threejsworker** [comment]
The examples of this pullrequest are now built and visible in threejsworker. To view them, go to the following link: 

 http://threejsworker.com/viewpullrequest.html#6404

**Comment by threejsworker** [comment]
The examples of this pullrequest are now built and visible in threejsworker. To view them, go to the following link: 

 http://threejsworker.com/viewpullrequest.html#6404

**Comment by threejsworker** [comment]
The examples of this pullrequest are now built and visible in threejsworker. To view them, go to the following link: 

 http://threejsworker.com/viewpullrequest.html#6404

**Comment by threejsworker** [comment]
The examples of this pullrequest are now built and visible in threejsworker. To view them, go to the following link: 

 http://threejsworker.com/viewpullrequest.html#6404

**Comment by threejsworker** [comment]
The examples of this pullrequest are now built and visible in threejsworker. To view them, go to the following link: 

 http://threejsworker.com/viewpullrequest.html#6404

**Comment by threejsworker** [comment]
The examples of this pullrequest are now built and visible in threejsworker. To view them, go to the following link: 

 http://threejsworker.com/viewpullrequest.html#6404

**Pull request closed without merging**